app/cronjob/trailingTradeIndicator/step/get-trading-view.js

```diff
@@ -110,19 +111,21 @@ const retrieveTradingView = async (logger, symbols, interval) => {
         );
 
         lastTradingView[symbol] = result;
-        cache.hset(
-          'trailing-trade-tradingview',
-          symbol,
-          JSON.stringify({
-            request: {
-              symbol,
-              screener: 'CRYPTO',
-              exchange: 'BINANCE',
-              interval
-            },
-            result
-          })
-        );
+        errorHandlerWrapper(logger, 'Trading View', async () => {
```

Comment:
I think this is duplicated and not necessary as you already added an error handler on https://github.com/chrisleekr/binance-trading-bot/pull/448/files#diff-de7732f4bf4cbcb2eaa03bdbb3b32ecaca24cd695a3539a9eb07a0b5356b4fe3R114